src/TouristInfoCards.js

```diff
@@ -0,0 +1,50 @@
+const TouristInfoCards = () => {
+    return (
+      <div className="cards ">
+        <div className="card ">
+          <img src="https://www.visitscotland.com/blog/wp-content/uploads/2021/10/Park_Circus_and_Kelvingrove_Park.jpg.jpg" className="card-img-top" />
+          <div className="card-body">
+          <h2>Glasgow</h2>
+            <p>
+            loren
+            </p>
+            <a href="https://peoplemakeglasgow.com/" className="btn btn-dark">
+              More information
+            </a>
+            
+
+          </div>
+        </div>
```

Comment:
This is great! You do have three cards though that repeat the same code, could you extract this logic into a `TouristInfoCard` component?